packages/core-components/src/components/table/table-header/table-header.tsx

```diff
@@ -126,6 +186,9 @@ export class TableHeaderComponent {
         ) : (
           <slot></slot>
         )}
+        <div
```

Comment:
div should only be present if resize is true, so a conditional render is a better choice here